wg-ecosystem/meeting-notes/2020-01-09.md

```diff
@@ -0,0 +1,23 @@
+# Ecosystem WG meeting 2020-01-09
+
+Thursday, Jan 09, 2020 (9:00-10:00am PT)
+
+## Attendees
+
+* @hashimoto
+* @malept
+* @erick
+* @kilian
+* @felix
+
+## Agenda
+
+* Key Result status updates
+  * Increase adoption of Electron-provided tooling (@ckerr)
+  * Increase views on /docs (@felix)
+* [Open pull requests for the WG to review](https://github.com/pulls?q=is%3Apr+team-review-requested%3Aelectron%2Fwg-ecosystem+archived%3Afalse+is%3Aopen+)
+* [Open pull requests mentioning the WG in a comment](https://github.com/pulls?q=is%3Apr+team%3Aelectron%2Fwg-ecosystem+archived%3Afalse+is%3Aopen)
+
+## Action Items
+
+* Views on /docs might not be best measurement. Will experiment with "Was this document helpful?" button on the docs web page for more meaningful feedback from users (@kilian).
```

Comment:
> Will experiment with "Was this document helpful?" button on the docs web page

Sounds like a good idea. A lo-fi way to do this would be to add a link to `github.com/electron/electronjs.org/issues/new?title=some-api&body=Here's my feedback`

Reply:
During the meeting, we discussed using [Google Analytics events](https://support.google.com/analytics/answer/1033068?hl=en).

Reply:
Yeah, I was gonna draft something to get:

* quantitative feedback with thumbs up/down using GA events
* qualitative feedback when someone gives a thumb down ("what can we do to improve)

Using GH issues for the latter one is a great idea @zeke! :)